src/components/FeatureFlags/feature-flags.json

```
@@ -271,6 +271,26 @@
             "defaultExistingProject": true
           }
         ]
+      },
+      "respectPrimaryKeyAttributesOnConnectionField": {
+        "description": "Changes the behaviour of relational directives on model type with custom primary key to generate connection fields, which respect the attributes of primary key and its sort key fields. The changes include naming convention, number of connected fields and their field types. In addition, this changes the code generation for datastore models of all platforms to enable the custom primary key feature.",
```

```suggestion
        "description": "Changes the behavior of relational directives on model types that contain a custom primary key by generating connection fields that respect the attributes of the primary key and sort key fields. The changes include naming convention, the number of connected fields, and their field types. In addition, this changes the code generation for DataStore models of all platforms to enable the custom primary key feature.",
```
nit: potential rewording